README.md

```diff
@@ -97,27 +115,26 @@ structs are defined in a typelibrary like this:
 
 ```json
 {
-	"module" : "example",
 	"types" : {
-		"my_type" : {
+		"data_t" : {
 			"members" : [
-				{ "name" : "integer", "type" : "uint32" },
-				{ "name" : "string",  "type" : "string" },
-				{ "name" : "array",   "type" : "uint16[]" }
+				{ "name" : "id", "type" : "string" },
+				{ "name" : "x", "type" : "fp32" },
+				{ "name" : "y", "type" : "fp32" },
+				{ "name" : "array", "type" : "uint32[]" }
 			]
 		}
 	}
 }
 ```
 
-Save above code to example.tld and run this through `dltlc` to generate C-headers and
-a typelibrary-definition-file, we will call it example.bin.
+Save above code to example.tld and run this through `dltlc` to generate an optional C-header and
+a `tld` file called example_tld.bin. `dltlc` must be run twice, like so.
```

Comment:
IMHO .tlb (type library binary) is better... Who decided to call this .bin ;)

Reply:
That's a good name!